build.sbt

```diff
@@ -28,12 +28,12 @@ libraryDependencies ++= Seq(
   "org.codehaus.janino" % "janino" % "2.7.8",
   "com.typesafe.slick" %% "slick" % "3.0.0",
   "com.zaxxer" % "HikariCP" % "2.3.3",
+  "org.hsqldb" % "hsqldb" % "2.3.2",
   "mysql" % "mysql-connector-java" % "5.1.35",
   //---------- Test libraries -------------------//
   "io.spray" %% "spray-testkit" % sprayV % Test,
   "org.scalatest" %% "scalatest" % "2.2.5" % Test,
   "com.typesafe.akka" %% "akka-testkit" % akkaV % Test,
-  "org.hsqldb" % "hsqldb" % "2.3.2" % "test",
   "org.liquibase" % "liquibase-core" % "3.3.5" % "test"
```

Comment:
I'm using Slick's database creation, instead of adding the liquibase binary as a runtime dependency. The `SlickDataAccessSpec` tests hsqldb against both creation methods, with an optional mysql run if one has the database setup locally.